### DISCUSSION TITLE
أضفت تعديلا بسيطا على التصميم

### DISCUSSION THREAD

**Discussion by WatheqAlshowaiter** [UX-Writing/Content-Style-Guide-in-Arabic#4] [General]
عند العرض على الأجهزة الصغير. يظهر العنوان ملاصقا لزر البرجر على اليمين كما في الصورة

<div dir="rtl">
<img src="https://user-images.githubusercontent.com/24838274/175891316-8411a3cb-9def-4793-a4d8-b31629d3be5e.png"
alt="صورة الخطأ"
 />
</div>
<br /><br />

أضفت القليل من المساحة لحل هذه المشكلة. ستظر كما في الشكل التالي
![Screen Shot 2022-06-27 at 11 09 17](https://user-images.githubusercontent.com/24838274/175891664-ebac02f3-3cd1-4441-bad8-5819f3863231.png)

أرجو سحب التحديثات إلى السيرفر عن طريق أمري
```
git pull 
yarn build
```

**Comment by MidooDj** [comment]
> عند العرض على الأجهزة الصغير. يظهر العنوان ملاصقا لزر البرجر على اليمين كما في الصورة
> 
> ![صورة الخطأ](https://user-images.githubusercontent.com/24838274/175891316-8411a3cb-9def-4793-a4d8-b31629d3be5e.png)
> 
> 
> أضفت القليل من المساحة لحل هذه المشكلة. ستظر كما في الشكل التالي ![Screen Shot 2022-06-27 at 11 09 17](https://user-images.githubusercontent.com/24838274/175891664-ebac02f3-3cd1-4441-bad8-5819f3863231.png)
> 
> أرجو سحب التحديثات إلى السيرفر عن طريق أمري
> 
> ```
> git pull 
> yarn build
> ```

شكرًا جزيلًا تم  اعتماد التعديل
